docs/pi/pi-e/_category_.json

```
@@ -0,0 +1,9 @@
+{
+  "label": "瑞莎 Pi E",
```

```suggestion
  "label": "ROCK Pi E",
```
产品名还是ROCK Pi E，只是我们分类成Radxa E系列

---

产品定义成了 ROCK Pi 系列